lighthouse-core/config/dobetterweb.json

```diff
@@ -49,6 +51,13 @@
         }
       }
     }, {
+      "name": "Using modern CSS features",
+      "audits": {
+        "uses-new-flexbox": {
```

Comment:
should this be called `'no-old-flexbox'`? or something for the expected result to be false